mlir/python/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Copyright (c) 2025.
+
+[project]
+name = "mlir-python-bindings"
+version = "0.0.1"
+requires-python = ">=3.8,<=3.13"
+[project.urls]
+Homepage = "https://github.com/llvm/llvm-project"
+Discussions = "https://discourse.llvm.org/"
+"Issue Tracker" = "https://github.com/llvm/llvm-project/issues?q=is%3Aissue%20state%3Aopen%20label%3Amlir%3Apython%20"
+"Source Code" = "https://github.com/llvm/llvm-project/tree/main/mlir/python"
+
+[build-system]
+requires = [
+    "scikit-build-core==0.10.7",
+    "typing_extensions==4.12.2",
+    "nanobind>=2.4, <3.0",
+    "numpy>=1.19.5, <=2.1.2",
+    "pybind11>=2.10.0, <=2.13.6",
```

Comment:
Is this due to some of the in repo dialects still using pybind? (downstream folks obviously weren't asked to switch etc)